jl4-lsp/src/LSP/L4/Actions.hs

```diff
@@ -49,12 +49,9 @@ topDeclToCompletionItem name = \case
          (_, unrollForall -> Fun {}) -> CompletionItemKind_Function
          _ -> CompletionItemKind_Constant
       }
-  KnownType kind _args tydec ->
+  KnownType kind _args _tydec ->
     Just (defaultTopDeclCompletionItem (typeFunction kind))
-      { CompletionItem._kind = Just $ case tydec of
-          RecordDecl {} -> CompletionItemKind_Struct
-          EnumDecl {} -> CompletionItemKind_Enum
-          SynonymDecl {} -> CompletionItemKind_Reference
+      { CompletionItem._kind = Just CompletionItemKind_Class
```

Comment:
This was not really accurate in my opinion, this was just a type constructor of some kind, and it makes the impl more complicated, so I removed it :)